### PR TITLE
Fix code format in iSCSI Multipath feature page

### PR DESCRIPTION
**Pull request by rollandf**
Changes proposed in this pull request:

- Fix code format

- Minor phrasing fixes

I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): @rollandf

This pull request needs review by: @mureinik

### CODE REVIEW

File: source/develop/release-management/features/storage/iscsi-multipath.html.md [hunk 1]
```diff
@@ -8,10 +8,10 @@ wiki_revision_count: 28
 wiki_last_updated: 2014-12-15
 feature_name: Configure iSCSI Multipathing
 feature_modules: storage
-feature_status: Released
+feature_status: Released in oVIrt 3.4
```
Comment (mureinik) [inline source/develop/release-management/features/storage/iscsi-multipath.html.md]: Typo - should be "oVirt"

File: source/develop/release-management/features/storage/iscsi-multipath.html.md [hunk 2]
```diff
@@ -8,10 +8,10 @@ wiki_revision_count: 28
...
 ---
 
-# i SCSI-Multipath
+# iSCSI-Multipath
```
Comment (mureinik) [inline source/develop/release-management/features/storage/iscsi-multipath.html.md]: Let's drop the "-" while we're at it?

File: source/develop/release-management/features/storage/iscsi-multipath.html.md [hunk 3]
```diff
@@ -32,11 +32,11 @@ Up until now the iSCSI connection between the Host and the iSCSI Storage Domain
 
 ### Detailed Description
 
-We introduce new managed entity, iSCSI Bond which grouping networks and storage targets reachable via these networks.
+We introduce a new managed entity, iSCSI Bond which groups networks and storage targets reachable via these networks.
```
Comment (mureinik) [inline source/develop/release-management/features/storage/iscsi-multipath.html.md]: While we're at it, who are "we"? How about "This feature introduces..."

Comment (rollandf) [inline source/develop/release-management/features/storage/iscsi-multipath.html.md]: done

File: source/develop/release-management/features/storage/iscsi-multipath.html.md (hunk 1 above)
Comment (rollandf) [inline source/develop/release-management/features/storage/iscsi-multipath.html.md]: done

**Comment by mureinik** [comment]
@rollandf Can you please rebase so this can be merged?

**Comment by rollandf** [comment]
Rebased

**Pull request merged**